constants.php

```diff
@@ -2,25 +2,26 @@
 //NB: Do not edit; use ./constants.local.php instead.
 
 //<Not customisable>
-define('FRESHRSS_MIN_PHP_VERSION', '7.2.0');
-define('FRESHRSS_VERSION', '1.21.1-dev');
-define('FRESHRSS_WEBSITE', 'https://freshrss.org');
-define('FRESHRSS_WIKI', 'https://freshrss.github.io/FreshRSS/');
-
-define('APP_NAME', 'FreshRSS');
-
-define('FRESHRSS_PATH', __DIR__);
-define('PUBLIC_PATH', FRESHRSS_PATH . '/p');
-define('PUBLIC_TO_INDEX_PATH', '/i');
-define('INDEX_PATH', PUBLIC_PATH . PUBLIC_TO_INDEX_PATH);
-define('PUBLIC_RELATIVE', '..');
-define('LIB_PATH', FRESHRSS_PATH . '/lib');
-define('APP_PATH', FRESHRSS_PATH . '/app');
-define('I18N_PATH', APP_PATH . '/i18n');
-define('CORE_EXTENSIONS_PATH', LIB_PATH . '/core-extensions');
-define('TESTS_PATH', FRESHRSS_PATH . '/tests');
+const FRESHRSS_MIN_PHP_VERSION = '7.2.0';
+const FRESHRSS_VERSION = '1.21.1-dev';
```

Comment:
This introduced a regression for
https://github.com/FreshRSS/FreshRSS/blob/df56d3b3bfe76079c8572a5abf6209af3182bf1b/Docker/hooks/build#L4
Cf. https://github.com/FreshRSS/FreshRSS/pull/5808#issuecomment-1793710791
I will fix